miniapp/build.gradle

```diff
@@ -95,15 +95,21 @@ dokka {
 }
 
 apply from: '../config/publish/android.gradle'
-publishing {
-    publications {
-        MiniApp(MavenPublication, androidArtifact())
+afterEvaluate {
```

Comment:
Why do we need `afterEvaluate` block here 🤔 ?

Reply:
yeah that looks surprising - if we didn't need it before, why do we need it now?

Reply:
AGP added built-in support for maven publishing of android artifacts, so we've switched to using that for the android-buildconfig repo. 

Previously, we were using [this plugin](https://github.com/wupdigital/android-maven-publish) which allows you to set the android components during Gradle's evaluation phase. However, with AGP's publishing plugin, the android components can't be accessed until `afterEvaluate` ([see here](https://developer.android.com/studio/build/maven-publish-plugin)).